README.md

```diff
@@ -1,186 +1,149 @@
+python eval_segmentation.py run_crf=True model_paths=[$CHKPTS_ROOT"final_only_dino_city_01-23_14:55:24/epoch2-step1199.ckpt"] run_name="jan24_dino_city_crf"
+python eval_segmentation.py run_crf=True model_paths=[$CHKPTS_ROOT"final_only_dino_pots_01-24_15:37:40/1809-cluster_miou46.57.ckpt"] run_name="jan24_dino_pots_crf"
 ```
-dataset_name
-|── imgs
-|   ├── train
-|   |   |── unique_img_name_1.jpg
-|   |   └── unique_img_name_2.jpg
-|   └── val
-|       |── unique_img_name_3.jpg
-|       └── unique_img_name_4.jpg
-└── labels
-    ├── train
-    |   |── unique_img_name_1.png
-    |   └── unique_img_name_2.png
-    └── val
-        |── unique_img_name_3.png
-        └── unique_img_name_4.png
-```
+### 3. Reproducing Figures 3 and 4
 
-Next in [`STEGO/src/configs/train_config.yml`](src/configs/train_config.yml) set the following parameters:
+In section 4 of the paper, we benchmark the performance of STEGO and traditional dimensionality reduction techniques across various target dimensions. Figure 4 mainly shows the performances without applying CRF to paint an undistorted picture of the underlying feature projection method. Hence, we must first generate the non-CRF results for STEGO and the DINO baseline.
 
-```yaml
-dataset_name: "directory"
-dir_dataset_name: "dataset_name"
-dir_dataset_n_classes: 5 # This is the number of object types to find
+```bash
+python eval_segmentation.py run_crf=False model_paths=[$MODELS_DIR"cocostuff27_vit_base_5.ckpt"] run_name="jan12_repro_coco"
+python eval_segmentation.py run_crf=False model_paths=[$MODELS_DIR"cityscapes_vit_base_1.ckpt"] run_name="jan12_repro_city"
+python eval_segmentation.py run_crf=False model_paths=[$MODELS_DIR"potsdam_test.ckpt"] run_name="jan12_repro_pots"
+python eval_segmentation.py run_crf=False model_paths=[$CHKPTS_ROOT"final_only_dino_coco_01-23_14:55:24/epoch0-step6399.ckpt"] run_name="jan24_dino_coco"
+python eval_segmentation.py run_crf=False model_paths=[$CHKPTS_ROOT"final_only_dino_city_01-23_14:55:24/epoch2-step1199.ckpt"] run_name="jan24_dino_city"
+python eval_segmentation.py run_crf=False model_paths=[$CHKPTS_ROOT"final_only_dino_pots_01-24_15:37:40/1809-cluster_miou46.57.ckpt"] run_name="jan24_dino_pots"
 ```
 
-If you want to train with cropping to increase spatial resolution run our [cropping utility](src/crop_datasets.py).
-
-Finally, uncomment the custom dataset code and run `python precompute_knns.py`
- from `STEGO\src` to generate the prerequisite KNN information for the custom dataset.
- 
-You can now train on your custom dataset using:
-```shell script
-python train_segmentation.py
+Now we train the STEGO segmentation head and the other dimensionality reduction techniques for each dataset. For STEGO, we simply execute the provided `train_segmentation.py` script with different target dimensions to ensure maximum reproducibility.
+
+```bash
+# COCOSTUFF
+for dim in 90 768 384 192 100 96 48 24 12 6 3:
```

Comment:
After a bash for loop, there is a `;`, not a `:`, please update all of them :)

Reply:
ah actually, if you do it line-by-line there is no need to include the `;`either, but the `:` need to go  :)